.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
Dependencies are entirely self-contained, so I don't think this last sentence is needed.

```suggestion
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
```